### PR TITLE
WebUI: Fix adding member manager for groups and host groups

### PR DESCRIPTION
**Pull request by serg-cymbaluk**
- fix API method call for adding member manager
- fix regressions in host group associated tables

Ticket: https://pagure.io/freeipa/issue/8123

### CODE REVIEW

**Comment by tiran** [comment]
The patch fixes the add/remove member manager feature.

File: ipatests/prci_definitions/temp_commit.yaml [hunk 1]
```diff
@@ -57,14 +57,17 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-latest/temp_commit:
-    requires: [fedora-latest/build]
+  fedora-31/test_webui_host_net_groups:
+    requires: [fedora-30/build]
```
Comment (netoarmando) [inline ipatests/prci_definitions/temp_commit.yaml]: Required job must be `fedora-latest/build`

**Comment by serg-cymbaluk** [comment]
@tiran 
WebUI tests passed.
Deleting temp commit...

**Comment by serg-cymbaluk** [comment]
master:

* c0b0c6b4b598acd7a867594d91b7f7cff47d2e5e WebUI: Fix adding member manager for groups and host groups

**Pull request closed without merging**